smart_proxy_container_gateway.gemspec

```diff
@@ -15,6 +15,7 @@ Gem::Specification.new do |s|
   s.license = 'GPLv3'
 
   s.required_ruby_version = '~> 2.5'
```

Comment:
If you're updating it in CI, shouldn't this also be updated to match?

Reply:
Thanks, good catch! Should be fixed on next push.